drgn_tools/sys.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+"""
+Helper to view sysinfo data
+"""
+import argparse
+import datetime
+import time
+from typing import Any
+from typing import Dict
+
+from drgn import Architecture
+from drgn import Program
+from drgn import ProgramFlags
+from drgn.helpers.linux import for_each_task
+from drgn.helpers.linux.sched import loadavg
+
+from drgn_tools.corelens import CorelensModule
+from drgn_tools.cpuinfo import aarch64_get_cpu_info
+from drgn_tools.cpuinfo import x86_get_cpu_info
+from drgn_tools.mm import totalram_pages
+from drgn_tools.table import print_dictionary
+from drgn_tools.util import human_bytes
+
+
+def load_average(prog: Program) -> str:
```

Comment:
Given that `loadavg` already exists, and the function of this is to simply turn it into a string that looks like what you want, a better name would be `loadavg_str`